src/demo/index.php

```diff
@@ -65,6 +65,9 @@ function gtag() { dataLayer.push(arguments); }
                 <label for="size">Font size</label>
                 <input class="param" type="number" id="size" name="size" placeholder="20" value="20">
 
+                <label for="duration">Print duration</label>
```

Comment:
Specify that the unit is milliseconds?
```suggestion
                <label for="duration">Print duration (ms)</label>
```